src/common/initializable-state/create.test.ts

```diff
@@ -24,10 +24,11 @@ describe("InitializableState tests", () => {
 
```

Comment:
This entire commit is very interesting, and introduces what could be repeatable patterns for benefit of many other scenarios.